src/bloaty.cc

```diff
@@ -1940,6 +1959,8 @@ USAGE: bloaty [OPTION]... FILE... [-- BASE_FILE...]
   --list-sources     Show a list of available sources and exit.
   --source-filter=PATTERN
                      Only show keys with names matching this pattern.
+  --not-source-filter=PATTERN
```

Comment:
I'm leaning towards `--exclude-source-filter`. It matches the rest of the PR's terminology better.